src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -138,60 +149,107 @@ const TableHeader: React.FC<TableHeaderProps> = ({
+          event.severity = "warning";
+          event.addMetadata("context", context);
+        });
+      });
+    },
+    [tableOfUser?.reference, tables, user, venueName]
+  );
+
   return (
-    // @debt move all style into TableHeader.scss
-    <div className="row no-margin at-table table-header">
-      <div className="header">
-        <div className="back-button-container">
-          <button
-            type="button"
-            title={"Leave " + seatedAtTable}
-            className="back-button"
-            id="leave-seat"
-            onClick={leaveSeat}
-          >
-            <FontAwesomeIcon
-              className="back-button-icon"
-              icon={faChevronLeft}
-              size="xs"
-            />
-            Leave table
-          </button>
-        </div>
-        <div className="table-title-container">
-          <div className="private-table-title">
-            {tableTitle}
-
-            {tableCapacity && (
-              <span className="private-table-seats-left">
-                {numberOfSeatsLeft} seats left
-              </span>
-            )}
-          </div>
-          {tableOfUser && tableOfUser.subtitle && (
-            <div className="private-table-subtitle">{tableOfUser.subtitle}</div>
-          )}
-        </div>
-        <div className="lock-button-container">
+    <div className="row table-header">
+      <div className="table-header__leave-table">
+        <button
+          type="button"
+          title={"Leave " + seatedAtTable}
+          className="table-header__leave-button"
+          id="leave-seat"
+          onClick={leaveSeat}
+        >
           <FontAwesomeIcon
-            className="lock-table-icon"
-            icon={isCurrentTableLocked ? faLock : faLockOpen}
-            size="sm"
+            className="table-header__leave-button--icon"
+            icon={faChevronLeft}
+            size="xs"
           />
-          <div className="lock-table-checbox-indication">
-            {isCurrentTableLocked ? "Table Locked" : "Lock Table"}
+          Leave table
+        </button>
+      </div>
+
+      <div className="table-header__topic-info">
+        <div className="row table-header__topic">
+          <div>{tableTitle}</div>
+
+          <div className="table-header__edit-topic-button" onClick={show}>
+            <FontAwesomeIcon icon={faPen} />
           </div>
-          <label className="switch">
-            <input
-              type="checkbox"
-              className="switch-hidden-input"
-              checked={!!isCurrentTableLocked}
-              onChange={toggleIsCurrentTableLocked}
-            />
-            <span className="slider" />
-          </label>
         </div>
+
+        {tableCapacity && (
+          <span className="table-header__seats-left">
+            {numberOfSeatsLeft} seats left
+          </span>
+        )}
+
+        {tableOfUser && tableOfUser.subtitle && (
+          <label>{tableOfUser.subtitle}</label>
```

Comment:
Why are we using `label` here? Semantically this relates to forms?

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
My intention from the beginning was to add form in this component. Which is now a thing. Fixed in https://github.com/sparkletown/sparkle/pull/1377/commits/c194508b129176d4c048ebb810e9ea4c59f60865

Reply:
I'm not seeing how the changes in that commit relate to using `label` here? Can you please explain deeper.

Reply:
Ahh, that's right. The form is in another component. I will change this to `div`

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/cd551774283c15520bf81075522602383141fc21